frontend/components/form/StatusBar.tsx

```diff
@@ -43,7 +43,11 @@ const StatusBar = ({ status }: iStatusBarProps) => {
 
   return (
     <>
-      <Group horizontal style={{ margin: '0 0.5rem' }}>
+      <Group
+        horizontal
+        className="flex items-center"
+        style={{ margin: '0 0.5rem' }}
+      >
```

Comment:
why not use tailwind to do the margin?

Reply:
Sounds good. A lot of stuff is just temporary changes while I move everything to Tailwind.